### DISCUSSION TITLE
map과 flatMap의 차이

### DISCUSSION THREAD

**Discussion by unanchoi** [Katchup-dev/Katchup-server#30] [General]
@yeseul106 
https://github.com/Katchup-dev/Katchup-server/pull/27#discussion_r1197272300

https://madplay.github.io/post/difference-between-map-and-flatmap-methods-in-java

**Comment by yeseul106** [comment]
오와 감쟈합니다 !!!! 똑순이 인텔리제이를 이해해보겠어요. ㅎ_ㅎ
